src/scenes/CreateEvent/store.js

```diff
@@ -503,6 +510,7 @@ export default class CreateEventStore {
         this.resultSetting.startTime.toString(),
         this.resultSetting.endTime.toString(),
         escrowAmountSatoshi,
+        this.language
```

Comment:
dont pass the string of the language (English, Korean, Chinese). use the language code: `en-US`, `zh-Hans-CN`, `ko-KR`. this should match the locale language codes.